### PR TITLE
bug fix for equalized learning rate

### PR DESCRIPTION
**Pull request by SaoYan**
Should set bias=False in nn.Conv2d, nn.Linear, etc.

Reference: https://github.com/tkarras/progressive_growing_of_gans/blob/master/networks.py#L48

### CODE REVIEW

File: pro_gan_pytorch/CustomLayers.py [hunk 1]
```diff
@@ -19,15 +19,16 @@ def __init__(self, c_in, c_out, k_size, stride=1, pad=0, initializer='kaiming',
         :param bias: whether to use bias or not
         """
         super(_equalized_conv2d, self).__init__()
-        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=True)
+        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=False)
```
Comment (akanimax) [inline pro_gan_pytorch/CustomLayers.py:22]: Ah right! It was set to `True` here and in `Linear`. In `deconv` it was already false. Thanks for the fix! :+1:

File: pro_gan_pytorch/CustomLayers.py [hunk 2]
```diff
@@ -19,15 +19,16 @@ def __init__(self, c_in, c_out, k_size, stride=1, pad=0, initializer='kaiming',
...
         if initializer == 'kaiming':
             th.nn.init.kaiming_normal_(self.conv.weight, a=th.nn.init.calculate_gain('conv2d'))
         elif initializer == 'xavier':
             th.nn.init.xavier_normal_(self.conv.weight)
 
         self.use_bias = bias
-
-        self.bias = th.nn.Parameter(th.FloatTensor(c_out).fill_(0))
+        
+        if self.use_bias:
```
Comment (akanimax) [inline pro_gan_pytorch/CustomLayers.py:30]: Hmmm... Cool :+1: This should reduce some GPU memory in case you don't need affine convolutions.

File: pro_gan_pytorch/CustomLayers.py [hunk 3]
```diff
@@ -164,8 +174,7 @@ def __init__(self, in_channels, use_eql):
             self.conv_2 = Conv2d(in_channels, in_channels, (3, 3), padding=1, bias=True)
 
         # Pixelwise feature vector normalization operation
-        self.pixNorm = lambda x: local_response_norm(x, 2 * x.shape[1], alpha=2,
-                                                     beta=0.5, k=1e-8)
```
Comment (akanimax) [inline pro_gan_pytorch/CustomLayers.py:164]: This implementation is correct, please substitute the values of `alpha`, `beta` and `k` in the formula [here](https://pytorch.org/docs/stable/nn.html#localresponsenorm). But I suppose explicit implementation works too. Cool :+1:
